src/sql/workbench/services/query/common/queryManagement.ts

```diff
@@ -149,6 +150,10 @@ export class QueryManagementService implements IQueryManagementService {
 		return this._queryRunners.get(uri);
 	}
 
+	public getProviderIdForUri(uri: string): string {
```

Comment:
nit: the forwarded method is named `getProviderIdFromUri` but this one is `getProviderIdForUri`.  I'd recommend either make consistent or simply make this one `getProviderId`.